4ian/GDevelop
# Renovate home page 🪚

Change start page to:
- Display starters for new users
- Display new tutorials
- Display latest games made with GDevelop
- Display button to open user's profile

Also:
- Move folder selection to another dialog that will host all the future project settings (project name for example).
- Remove Starters tab from the "Create Project" dialog
- Rename Startpage to Homepage
- Add split menu to `Open Project` button with recent projects

Screenshots:
- Desktop:
  ![image](https://user-images.githubusercontent.com/32449369/145436233-e302194c-02c9-46e6-9542-3c7094c4c962.png)

- Mobile
  ![image](https://user-images.githubusercontent.com/32449369/145436364-f84b93d3-7608-400e-a63a-588f120b1431.png)

- Click on showcased game
  ![image](https://user-images.githubusercontent.com/32449369/144892891-4bb74c1a-d664-490d-a78f-2c6973c4fe2a.png)

- Recent files as split menu
  ![image](https://user-images.githubusercontent.com/32449369/145195729-8b2e61a6-8c6c-4329-93aa-ecaf4297296e.png)


- Check Storybook for error handling:
  - On Carousel : https://gdevelop-storybook.s3.amazonaws.com/refurbish-homepage-for-good/latest/index.html?path=/story/carousel--with-error
  - On Home Page : https://gdevelop-storybook.s3.amazonaws.com/refurbish-homepage-for-good/latest/index.html?path=/story/homepage--with-server-side-errors
